routing.rst

```diff
@@ -12,6 +12,21 @@ URL of a page from ``/blog`` to ``/news``? How many links would you need to
 hunt down and update to make the change? If you're using Symfony's router,
 the change is simple.
 
+Before using the annotations, be sure that you installed the annotations support :
```

Comment:
I'd say "If you want to use the annotations, make sure you have installed the `annotations` recipe:"